observability/prometheus-backend/src/main/scala/sttp/client3/prometheus/PrometheusBackend.scala

```diff
@@ -60,6 +65,54 @@ object PrometheusBackend {
     )
   }
 
+  /** Add, if not present, label pairs for "method" and "status" (if Response is non-empty). In other words, if the user
+    * already supplied those labels, leave them as-is.
+    * @param config
+    *   user-supplied PrometheusBackend base collector
+    * @param req
+    *   Request
+    * @param maybeResp
+    *   Optional response
+    * @return
+    *   BaseCollectorConfig sub-type. Note that PrometheusBackend#apply and PrometheusListener's constructor reference
+    *   the sub-types of BaseCollectorConfig.
+    */
+  def addLabelPairs(config: BaseCollectorConfig, req: Request[_, _], maybeResp: Option[Response[_]]): config.T = {
```

Comment:
Maybe a better name would be sth like `addMethodStatusLabels`? It would reflect more clearly what it does

Reply:
Good call - will add.

Reply:
Done - https://github.com/softwaremill/sttp/pull/1651/commits/d802a5d4fd2d74fc4a70d222488c2317e95c3f80